Dockerfile

```diff
@@ -148,6 +148,8 @@ RUN git clone https://github.com/ztombol/bats-support /bats/bats-support && \
 RUN wget https://github.com/mikefarah/yq/releases/latest/download/yq_linux_amd64 -O /usr/bin/yq && \
     chmod +x /usr/bin/yq
 
+RUN sed -i "s|v0.4.1-0.20230214201333-88ed8ca3307d|v0.7.0|g" /usr/local/go/src/go.mod
```

Comment:
What is this meant to do?

Reply:
Ah yes. I copied this from #503 to mitigate a container security issue being flagged. I actually left a comment on that PR for @rpothier asking whether this is meant to be temporary or... So I'm not sure yet basically ;)

Reply:
yes this is temporary. Added a grep command in #503 to force a script failure when the version changes so we don't forget to remove this.

Reply:
Great. I copied that here as well